tests/integration/mixer/policy/ratelimiting_test.go

```diff
@@ -88,7 +85,7 @@ func TestRateLimiting_DefaultLessThanOverride(t *testing.T) {
 				want429s, want429s/actualDuration)
 
 			// As rate limit is applied at ingressgateway itself, fortio should see the limits too.
-			want := math.Floor(want200s * 0.90)
+			want := math.Floor(want200s * 0.70)
```

Comment:
Changed it back to 90%... Looking at https://testgrid.k8s.io/istio-postsubmits-master#integ-mixer-k8s-postsubmit-tests&sort-by-flakiness= , DefaultLessThanOverride test has not been flaky...
(I just saw it flaky in my local environment and thought it could be because of https://github.com/istio/istio/issues/3028#issuecomment-442550779) 

For flake in FixedWindow, I saw similar logs in all the flakes from this test.. it's as if system needed more time to sync the rules.. in old framework also, we solved it by sending some initial traffic to boot the system...